pokete_data/map_data.py

```diff
@@ -1656,7 +1654,8 @@
             "tree_group_1": {
                 "x": 0,
                 "y": 0,
-                "txt": """()(()())))))((())(()))((((())))((((()))))))))(()())((()()))()))()()()())))(((())()))()())))(()((())))(())()())))()(()())
+                "txt": """
```

Comment:
This code produces a Coordinate error and a uneeded line at line of spaces at the start of the screen.

Reply:
Fixed in ee3504b7128cc58e2dcc0a451993ace913c5fc08